internal/cli/stream_server_logger.go

```diff
@@ -0,0 +1,28 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	"github.com/windmilleng/tilt/internal/logger"
+	"github.com/windmilleng/tilt/internal/stream"
+)
+
+type StreamWriter struct {
+	s stream.StreamServer
+}
+
+func (sw StreamWriter) Write(b []byte) (int, error) {
+	sw.s.Send(string(b))
+	return len(b), nil
+}
+
+func ContextWithStreamServerLogger(ctx context.Context) (ctx2 context.Context, close func(), err error) {
```

Comment:
maybe just delete the named params? they're not adding anything

Reply:
You mean drop the names of the return values and keep the return values themselves?
I added them because I figured the intent of `func()` isn't very clear without a name. I guess I can go the `context.CancelFunc` route.